model/interactive/OWNERS

```diff
@@ -0,0 +1,7 @@
+# See the OWNERS docs at https://s.apache.org/beam-owners
+
+reviewers:
```

Comment:
You might want to limit this list to people who are likely to maintain this path.

Reply:
Done